core/org.wso2.carbon.core.services/src/main/java/org/wso2/carbon/core/services/util/CarbonAuthenticationUtil.java

```diff
@@ -68,7 +68,8 @@ public static void onFailedAdminLogin(HttpSession httpSess, String username, int
         Date currentTime = Calendar.getInstance().getTime();
         SimpleDateFormat date = new SimpleDateFormat("'['yyyy-MM-dd HH:mm:ss,SSSZ']'");
 
-        String msg = "Failed Administrator login attempt \'" + username + "[" + tenantId + "]\' at "
+        //error msg to be displayed for an non-admin user login
+        String msg = "Non-Admin user login attempt \'" + username + "[" + tenantId + "]\' at "
```

Comment:
Don't we need to mention it's a failed attempt? Also do we print the correct message when there is a failed admin login attempt?